controllers/mattermost/mattermost/mattermost.go

```diff
@@ -402,23 +403,27 @@ func (r *MattermostReconciler) checkUpdateJob(
 		err = r.Resources.RestartMattermostUpdateJob(mattermost, job, baseDeployment, reqLogger)
 		if err != nil {
 			recStatus.ResourcesReady = false
-			return nil, recStatus, errors.Wrap(err, "failed to restart update job")
+			reqLogger.Error(err, "failed to restart update job")
+			return nil, recStatus, nil
```

Comment:
In this case, we should return an error as the restart of the update job failed.

Reply:
ok